Makefile

```diff
@@ -1,18 +1,23 @@
-.PHONY: help start test shell loaddata
+.PHONY: help start stop test shell flush loaddata
```

Comment:
A cleaner pattern would be to add the `.PHONY: foo` atop of every make target instead. Helps with not forgetting it.